### PR TITLE
Add csscomb as a beautifier 

### PR DESCRIPTION
**Pull request by muuvmuuv**
See: #136 

Noticed that CSScomb misses `csscomb` and `cosmiconfig` as a dependency. My bad, I'm not familiar with the NPM ecosystem and how they resolve packages between others. Fix will come soon @Glavin001

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -87,6 +87,7 @@
     },
     "dependencies": {
         "@unibeautify/beautifier-clang-format": "^0.2.0",
+        "@unibeautify/beautifier-csscomb": "^0.1.0",
```
Comment (Glavin001) [inline package.json]: Need to add `csscomb` as dependency, too.

File: package.json [hunk 2]
```diff
@@ -95,6 +96,7 @@
         "@unibeautify/beautifier-prettydiff": "^0.5.3",
         "@unibeautify/beautifier-sqlformat": "^0.0.1",
         "cosmiconfig": "^5.0.5",
+        "csscomb": "^4.2.0",
```
Comment (Glavin001) [inline package.json]: New information: https://github.com/Unibeautify/beautifier-csscomb/pull/12#issuecomment-425911302

Apologize for the confusion. I need to stop reviewing PRs so late at night 😜 .

File: package.json [hunk 3]
```diff
@@ -95,6 +95,7 @@
         "@unibeautify/beautifier-prettydiff": "^0.6.0",
         "@unibeautify/beautifier-sqlformat": "^0.0.1",
         "cosmiconfig": "^5.0.6",
+        "csscomb": "^4.2.0",
```
Comment (Glavin001) [inline package.json]: Error in Travis CI build:

```
src/beautifiers.ts:10:21 - error TS2307: Cannot find module '@unibeautify/beautifier-csscomb'.
```

Looks like `@unibeautify/beautifier-csscomb` is missing. Need to add it as dependency. 🎉

File: package.json [hunk 4]
```diff
@@ -87,15 +87,14 @@
     },
     "dependencies": {
         "@unibeautify/beautifier-clang-format": "^0.2.0",
-        "@unibeautify/beautifier-csscomb": "^0.1.0",
```
Comment (Glavin001) [inline package.json]: Oops looks like a mistake during merge conflict resolution -- deleted `@unibeautify/beautifier-csscomb` 😜 .

Comment (muuvmuuv) [reply package.json]: @Glavin001 isn't it resolved?

**Comment by stevenzeck** [comment]
@muuvmuuv can you rebase and do a fresh `npm install` and commit any changes to package-lock.json?  Also run `npm run vscode:prepublish` and commit both of those changes.

**Comment by muuvmuuv** [comment]
@stevenzeck done ✔️

**Comment by stevenzeck** [comment]
Thanks for contributing!

**Comment by Glavin001** [comment]
Published to v0.7.0

**Pull request merged**